report_pre_thesis/src/sections/experiments.tex

```diff
@@ -1,26 +1,115 @@
+	\item\label{enum:experiment4} Evaluating the impact on ICA model performance when using all five location datasets versus just one.
+	\item\label{enum:experiment5} Other models to compare the performance of the PLS1-SM and ICA models to other models.
+	% \item PLS outlier removal on ICA (?) instead of MAD.
+\end{enumerate}
 
 \noindent
-Given that the original authors did not perform experiments using alternative methods to demonstrate the efficacy of their chosen approach, this omission results in a lack of comprehensive understanding regarding the full potential of the pipeline's performance.
-While they did perform hyperparameter tuning, they did not conduct experiments using different outlier removal methods or alternative models.
-This raises questions about the optimality of the chosen methodology, as a comparative analysis with different methodologies could reveal superior approaches.
-Experimenting with alternative methods means that we can uncover which components contribute the most to the overall error and therefore would benefit the most from further research and development.
-Should a substitution of a component within the pipeline with an alternative method yield improved outcomes, it would indicate that the currently employed method represents a limitation in the overall pipeline, thus highlighting an area that necessitates enhancement.
+We have chosen these experiments based on the following reasons.
+For Experiment~\ref{enum:experiment1}, we are interested in evaluating the significance of our automated outlier removal process due to its differences from the original manual process.
+Additionally, we are interested in examining whether outlier removal is necessary for the PLS1-SM phase by comparing the results with and without outlier removal.
+
+For Experiment~\ref{enum:experiment2}, we want to evaluate the significance of the threshold values used in the outlier removal process by maintaining the threshold values from the second iteration of the outlier removal in the PLS1-SM training process.
+This is a more conservative than always updating the threshold values after each iteration.
+
+For Experiment~\ref{enum:experiment3}, we implement our best guess of the original authors' outlier removal process for the ICA phase, which is based on the Median Absolute Deviation (MAD).
+Then, we compare the results with and without MAD to evaluate the significance of the outlier removal step in the ICA phase.
+
+For Experiment~\ref{enum:experiment4}, we perform ICA using all five location datasets by averaging the results from location into a single dataset.
+By doing this, we will either obtain a more representative dataset for the sample, or we will end up losing information by averaging the results.
+
+For Experiment~\ref{enum:experiment5}, we want to compare the performance of the MOC model with other models.
+Specifically, we train an XGBoost model and an ANN model for comparison.
 
```

Comment:
I just read the report in the PDF and I think my confusion about the justifications comes from the fact that there is no clear connection between the "chosen experiments" section and the following experiment sections where we outline the reasons a little more. I think we need to work on tying these together a little more.